.travis/run_travis_job.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+# Fail immediately in case of errors and/or unset variables
+set -eu -o pipefail
+
+# Echo commands so that the progress can be seen in CI server logs.
+set -x
+
+# Detect installed Java
+export JAVA_HOME="$(java -XshowSettings:properties -version \
+    2>&1 > /dev/null |\
+    grep 'java.home' |\
+    awk '{print $3}')"
+
+export LD_LIBRARY_PATH="$(find -L ${JAVA_HOME} -type f -name libjvm.* | xargs -n1 dirname)"
+
+# Install clippy
+if rustc --version | grep -q "nightly"
+then
+    # Install nightly clippy
+    rustup component add clippy --toolchain=nightly || cargo install --git https://github.com/rust-lang/rust-clippy/ --force clippy
```

Comment:
Why are there two commands (||)?

Reply:
Clippy might not be available for a certain nightly release and this is an official recommendation to install it that way - https://github.com/rust-lang/rust-clippy#travis-ci. I believe it's totally OK because we allow nightly build to fail.